src/main/java/uk/gov/crowncommercial/dts/scale/cat/config/OAuth2Config.java

```diff
@@ -28,6 +28,7 @@ protected void configure(HttpSecurity http) throws Exception {
       authz
         // TODO: Update to correct roles when fixed in Auth server
         .antMatchers(HttpMethod.POST, "/tenders/projects/agreements").hasAuthority("[\"CAT_ADMINISTRATOR\",\"CAT_USER\"]")
+        .antMatchers(HttpMethod.POST, "/tenders/projects/*/events").hasAuthority("[\"CAT_ADMINISTRATOR\",\"CAT_USER\"]")
```

Comment:
I don't know how granular we want / need to be with these, going forward.. could combine into a single `/tenders/*` pattern for now I guess 🤷

Reply:
Yep - makes sense - I'll update that 👍